lasagne/objectives.py

```diff
@@ -1,35 +1,50 @@
+import theano
 import theano.tensor as T
 
 
-def mse(x, t):
+def mse(x, t, m):
     """Calculates the MSE mean across all dimensions, i.e. feature
      dimension AND minibatch dimension.
 
     :parameters:
         - x : predicted values
         - t : target values
+        - m : mask; None for no mask, or a 1-dimensional array, that
+            selects which rows in `x` and `t` should count towards the
+            result
```

Comment:
Checking the code, it correctly handles masks that are the same shape as `x` and `t`, and it correctly handles non-binary masks (something I use a lot). The docstring should be updated to reflect that.